### PR TITLE
Group by ciBuildId

### PR DESCRIPTION
**Pull request by Upgreydd**
thx to @pleszkowicz

- Adds Toggle for stacking builds by `ciBuildId`
- Nice formatting and minimizing would be done in next PR

Stack runs disabled:
image
Stack runs enabled:
image
Stack runs enabled and accordion expanded:
image

### CODE REVIEW

File: packages/dashboard/src/views/RunsView.tsx [hunk 1]
```diff
@@ -73,11 +75,30 @@ export function RunsView({
   if (!runFeed.runs.length) {
     return <div>No runs have started on this project.</div>;
   }
+
+  const groupedRuns: any[] = [];
+  data.runFeed.runs.forEach((run) => {
```
Comment (jhicken) [inline packages/dashboard/src/views/RunsView.tsx:80]: What do you do if all the runs for this group are not in this page of results?

Comment (jhicken) [reply packages/dashboard/src/views/RunsView.tsx:80]: I think this needs more thought. Particularly if we add filtering to these run results. Which I think we should.

**Comment by agoldis** [comment]
"Run" is actually a "group by ciBuildId", I think removing specs from runId hash would solve the problem for @Upgreydd. Will validate that nothing goes left tonight. More details in https://github.com/sorry-cypress/sorry-cypress/pull/181. 

--- 

- when creating a new run sc calculates a hash that is based on `ciBuildId + commit.sha + projectId + specs.join(' ')`. If two machines send identical parameters, they will be associated with the same run. To support the scenario above we need to get rid of specs list while building that hash.

- it tries to create a new run with runId, if creation fails - there's already an existing run, so we are returning this pre-existing run

File: packages/dashboard/src/views/RunsView.tsx (hunk 1 above)
Comment (Upgreydd) [reply packages/dashboard/src/views/RunsView.tsx:80]: it will be updated after clicking `Load more`

**Comment by Upgreydd** [comment]
Resolved in https://github.com/sorry-cypress/sorry-cypress/pull/189

**Comment by luengnat** [comment]
@Upgreydd  the accordion is still not in the build. Should we add the support for that as well?

**Pull request closed without merging**